models/DataObject/SelectOptions/Config/Dao.php

```diff
@@ -0,0 +1,160 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Model\DataObject\SelectOptions\Config;
+
+use Pimcore\Config;
+use Pimcore\Config\LocationAwareConfigRepository;
+use Pimcore\Model;
+
+/**
+ * @internal
+ *
+ * @property \Pimcore\Model\DataObject\SelectOptions\Config $model
+ */
+class Dao extends Model\Dao\PimcoreLocationAwareConfigDao
+{
+    public function configure(): void
+    {
+        $config = Config::getSystemConfiguration();
+
+        $storageConfig = $config['config_location']['select_options'];
```

Comment:
'select_options' can be added as a constant like https://github.com/pimcore/pimcore/blob/20cfe5c3685bf7c82af2011062a257d73c32e986/models/Asset/Image/Thumbnail/Config/Dao.php#L28, so that it will be more consistent.

Reply:
Ah, yes. The static analysis complained about an unused constant, so I removed it. I'll put it back.